specs/plugin-extensibility.md

```diff
@@ -20,15 +20,41 @@ Keys and associated certificates used for signing artifacts using Notation could
   * Notation MUST work with a plugin that implements a matching or lower minor version of the plugin contract. Notation SHALL NOT support using a plugin with higher version of plugin contract.
   * A plugin MUST support a single plugin contract version, per major version.
 
-Notation will invoke plugins as executable, pass parameters using command line arguments, and use standard IO streams to pass request/response payloads. This mechanism is used as Go language (used to develop [Notation Go library](https://github.com/notaryproject/notation-go)) does not have a [in built support](https://github.com/golang/go/issues/19282) to load and execute plugins that works across OS platforms. Other mechanisms like gRPC require every plugin to be implemented as a service/daemon.
+Notation will invoke plugins as executable, pass parameters using command line arguments, and use standard IO streams to pass request/response payloads.
+
+### Package a plugin
+
+Plugin publisher SHOULD follow these conventions to package and distribute plugin files:
```

Comment:
```suggestion
Plugin publisher SHOULD follow these conventions to package and release plugin files:
```

Reply:
Same as the comment above https://github.com/notaryproject/specifications/pull/292#discussion_r1471544005